tsl/src/nodes/decompress_chunk/decompress_chunk.h

```diff
@@ -92,6 +92,19 @@ typedef struct DecompressChunkPath
 	 */
 	bool perform_vectorized_aggregation;
 
+	/*
+	 * Columns that are used for vectorized aggregates. The list contains for each attribute -1 if
+	 * this is not an vectorized aggregate column or the Oid of the data type of the attribute.
+	 *
```

Comment:
```suggestion
```

Reply:
@fabriziomello Comment is adjusted. Thanks for the review.